aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -3506,5 +3508,63 @@ Tensor _weight_int8pack_mm_cpu(
   return C;
 }
 
+Tensor& _int_mm_out_cpu(const Tensor& self, const Tensor& mat2, Tensor& result) {
+  TORCH_CHECK(self.dim() == 2, __func__, ": Expected self to be of dimension 2 but got ", self.dim());
```

Comment:
`__func__`  is not standard. Better define a constexpr at the top. Also, these are user facing names. They should not use internal names of functions.

Reply:
Thanks. I have defined a string "int_mm_out_cpu" without the leading underscore.